lending-pool/src/pool/bad_debt.rs

```diff
@@ -245,67 +246,4 @@ mod tests {
             transfer_bad_debt_to_backstop(&e, &samwise);
         });
     }
-
-    /***** burn_backstop_bad_debt ******/
-
-    #[test]
-    fn test_burn_backstop_bad_debt() {
```

Comment:
was this test removal an accident? It appears to be the only one that tests `burn_backstop_bad_debt`